packages/core/src/endowmentsToolkit.js

```diff
@@ -130,6 +134,39 @@ function endowmentsToolkit({
     return targetRef
   }
 
+  /**
+   * @param {object} moduleNamespace
+   * @param {string} moduleId
+   * @param {LMPolicy.BuiltinPolicy} policyBuiltin
+   */
```

Comment:
- This appears to transform `moduleNamespace`.  Please note if it mutates `moduleNamespace` or not
- Adding a `@returns` is good practice wherever you can, but for public APIs, you especially want to declare the return type for performance reasons.